ui/components/app/account-menu/index.scss

```diff
@@ -116,6 +115,10 @@
     }
   }
 
+  &__refresh-button {
+    color: var(--Blue-400);
```

Comment:
I believe this can be `var(--color-primary-default)` to support our dark mode efforts